### ISSUE TITLE
Review and approve launch of the exported PDFs from "MHV Rx" team

### ISSUE THREAD

**Issue by vmenshutin-bylight** [department-of-veterans-affairs/va.gov-team#67990]
### What do you need support with?

Please review the following PDFs and approve launching with them.

These PDFs have been reviewed within https://github.com/department-of-veterans-affairs/va.gov-team/issues/66799. 

The following issues have been fixed:
* Empty < H4 > tags after each < H3 > of page
* Provide content for Subject fields in Metadata

More information could be found in the comments section here: https://jira.devops.va.gov/browse/MHV-50186

Other issues require major investigation and enhancements, which would influence both medical records and medications teams. These issues are sought to be addressed after the launch.

### When do you need this support by?

ASAP

### How urgent is this request?

Very urgent, this is blocking other critical work and is needed ASAP

### Supporting artifacts (if any)

[Non-VA-medications-details-Harry-Potter-10-19-2023_21829pm.pdf](https://github.com/department-of-veterans-affairs/va.gov-team/files/13046413/Non-VA-medications-details-Harry-Potter-10-19-2023_21829pm.pdf)
[VA-medications-details-Harry-Potter-10-19-2023_21618pm.pdf](https://github.com/department-of-veterans-affairs/va.gov-team/files/13046414/VA-medications-details-Harry-Potter-10-19-2023_21618pm.pdf)
[VA-medications-list-Harry-Potter-10-19-2023_22252pm.pdf](https://github.com/department-of-veterans-affairs/va.gov-team/files/13046415/VA-medications-list-Harry-Potter-10-19-2023_22252pm.pdf)

### Who's on your team?

Team name: vfs-mhv-medications
product owner: Raquel Robertson
Product manager: Tanner Bird
Designer: @coultonbunney-usds
FE engineer: @acrollet @vmenshutin-bylight
Product/team Slack channel: #[mhv-medications-rx]

### Do you have an accessibility champion on your team?

No, we don't have a champion (yet)

**Comment by SarahKay8** [comment]
@vmenshutin-bylight Everything else is good.  However, because I can't see the conversation that's happening in JIRA, could you summarize why these list(s) are still in paragraph tags?

![91B1E670-F8C4-4CB8-AA46-3DE70646AB98](https://github.com/department-of-veterans-affairs/va.gov-team/assets/102324990/0013ab18-5a11-4f8d-a883-425e9e2d8bdf)

**Comment by SarahKay8** [comment]
When a visual list isn't tagged as a list and is instead placed within a paragraph tag in a PDF, it has a lot of negative effects on AT users, _especially_ those who rely on screen readers. 

VO + Jaws uses tags and structure within a document to provide a meaningful reading order. When a list isnt tagged as such and is placed in  a paragraph tag, it loses its structural significance. Screen readers won't recognize it as a list, making it v difficult for users to understand that the content is part of a list.

Also - When lists are not correctly tagged, it disrupts the logical flow of the document and makes it harder for users to navigate efficiently.

I wouldn't consider this launch blocking, but it is a _huge_ a11y concern @vmenshutin-bylight

**Comment by SarahKay8** [comment]
cc: @acrollet

**Comment by acrollet** [comment]
@vmenshutin-bylight fwiw I do see that list markup is [supported by pdfkit](http://pdfkit.org/docs/accessibility.html)

**Comment by klawyer** [comment]
Thanks @SarahKay8! I am the OCTO Product lead on this effort. 

Decision: Our first launch is only to a few users (Phase 0), so we are going to move forward with that launch with this as-is. But, this will be fixed before we expand to a larger set of users (Phase 1).

**Comment by SarahKay8** [comment]
Thanks @klawyer! Good to close this out?

**Comment by SarahKay8** [comment]
Just an idea of how it reads. 

> When a visual list isn't tagged as a list and is instead placed within a paragraph tag in a PDF, it has a lot of negative effects on AT users, _especially_ those who rely on screen readers.
> 
> VO + Jaws uses tags and structure within a document to provide a meaningful reading order. When a list isnt tagged as such and is placed in a paragraph tag, it loses its structural significance. Screen readers won't recognize it as a list, making it v difficult for users to understand that the content is part of a list.
> 
> Also - When lists are not correctly tagged, it disrupts the logical flow of the document and makes it harder for users to navigate efficiently.
> 
> ### Video for reference (sound on)
>  20231020140952229.mp4 
> I wouldn't consider this launch blocking, but it is a _huge_ a11y concern @vmenshutin-bylight

Added a video for reference just in case
